ajax/libs/scribbletune/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "scribbletune",
+  "description": "Scribbletune is a JavaScript library that helps you construct musical ideas with JavaScript String & Array functions and export them as MIDI files.",
+  "filename": "scribbletune.min.js",
+  "author": {
+    "name": "Walmik",
+    "email": "walmik.deshpande@gmail.com"
+  },
```

Comment:
Let's add the homepage, what do you think?
`"homepage": "https://scribbletune.com/"`
Many thanks

Reply:
Thank you for your comments @sufuf3 I ll add the home page info now.

Reply:
I ve added it now @sufuf3 :)